tests/components/zwave_js/fixtures/device_diagnostics.json

```diff
@@ -20,7 +20,7 @@
   "custom_components": {
     "pyscript": {
       "version": "1.5.0",
-      "requirements": ["croniter==1.3.8", "watchdog==2.3.1"]
+      "requirements": ["croniter==1.3.8", "watchdog==6.0.0"]
```

Comment:
This is a test fixture that doesn't need updating. Please revert here.

Reply:
Good catch. Thank you!